common/metrics/defs.go

```diff
@@ -2796,6 +2797,7 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		TaskWriteThrottlePerTaskQueueCounter:      NewRollupCounterDef("task_write_throttle_count_per_tl", "task_write_throttle_count"),
 		TaskWriteLatencyPerTaskQueue:              NewRollupTimerDef("task_write_latency_per_tl", "task_write_latency"),
 		TaskLagPerTaskQueueGauge:                  NewGaugeDef("task_lag_per_tl"),
+		NoRecentPollerTasksPerTaskQueueCounter:    NewRollupCounterDef("no_poller_tasks_per_task_queue", "no_poller_tasks"),
```

Comment:
On second thought + discussion with Yimin, I think it's better to make this name consistent with the others `_per_tl`, and then later change them all at once (with a new partition tag)